x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -2958,6 +2960,8 @@ public void testValidJoinPattern() {
     }
 
     public void testInvalidJoinPatterns() {
+        assumeTrue("LOOKUP JOIN requires corresponding capability", EsqlCapabilities.Cap.JOIN_LOOKUP_V12.isEnabled());
```

Comment:
Please let me know if it is more preferable to rely on `Build.current().isSnapshot()` instead since `JOIN_LOOKUP_V12` version is frequently incremented.

Reply:
I think this is fine/preferable. We have one main capability `JOIN_LOOKUP_V12` and all related caps should be enabled or disabled based on this.